README.md

```diff
@@ -274,6 +274,13 @@ A user can set permissions via `nfs4_setfacl` on a collection or data object if
 - `chmod` is currently implemented as a `NOP` and will return `0`.
 - NFSRODS currently reports disk free (`df -a`) as `0` to avoid being misleading to other programs.
 
+## Limitations
+### Limitation 1: Write operations do not trigger replication on a replication resource
+This is primarily due to NFSRODS being at the mercy of the client-side application. NFSRODS cannot make assumptions about what a client-side application will do.
```

Comment:
i'd add ... "with regards to multiple connections and streams into a particular data object".

Reply:
you're saying append to the second sentence, right?

Reply:
yes... 
"...will do with regards..."

Reply:
singular regard sounds better...

Reply:
updated.